include/filesystem/dstandardpaths.h

```diff
@@ -29,14 +29,33 @@ class LIBDTKCORESHARED_EXPORT DStandardPaths
     static QString findExecutable(const QString &executableName, const QStringList &paths = QStringList());
     static void setMode(Mode mode);
 
+    /**
+     * @brief About xdg dir, view it in https://gitlab.freedesktop.org/xdg/xdg-specs/
+     */
     enum class XDG {
+        /*
+         * @brief DataHome, usually is ~/.local/share, also can be defined by ${XDG_DATA_HOME}, where storages the data of applications
+         */
         DataHome,
+        /*
+         * @brief ConfigHome, usually is ~/.config, can be defined by ${XDG_CONFIG_HOME}, where storages the config of applications
+         */
         ConfigHome,
+        /*
+         * @brief CacheHome, usually is ~/.cacee, can be defined by ${XDG_CACHE_HOME}, where storages caches, can be always cleared
+         */
         CacheHome,
+        /*
+         * @brief Where temp files or sock files always be putted in, like sddm.sock. It is unique per session. It is always /run/user/${uid} or ${XDG_RUNTIME_DIR},
+         */
         RuntimeDir,
 #if DTK_VERSION < DTK_VERSION_CHECK(6, 0, 0, 0)
-        RuntimeTime [[deprecated("Use RuntimeDir Instead")]] = RuntimeDir
+        RuntimeTime [[deprecated("Use RuntimeDir Instead")]] = RuntimeDir,
 #endif
+        /*
+         * @brief where history file and log file should in. It is induced because users do not want to mix their config files and state files. It is always be ~/.local/state, or defined by XDG_STATE_HOME
```

Comment:
```suggestion
         * @brief where history file and log file should be. It is induced because users do not want to mix their config files and state files. It is always ~/.local/state, or defined by XDG_STATE_HOME
```